### ISSUE TITLE
[bug] 2024.1.3 doesn't work together with Lombok.

### ISSUE THREAD

**Issue by EotT123** [manifold-systems/manifold-ij#34]
When using new version 2024.1.3, Lombok isn't working anymore (Lombok imports cannot be resolved). Building with maven does work. I don't see any error messages. 
Also many IntelliJ features are working anymore, including but not limited to: 
* browse type hierarchy
* browse call hierarchy
* go to implementation
* go to declaration
* ...

**Comment by rsmckinney** [comment]
What version of IJ are you using? Did you build 2024.1.3?

**Comment by rsmckinney** [comment]
Found the issue, I assume you are using 2024.1.x, right?  Anyhow a fix is on the way for tomorrows IJ plugin release.  Thanks for the heads up,

**Comment by EotT123** [comment]
Yes, I built version 2024.1.3 myself to test if it resolved the problem. I've now built the new version (2024.1.4), and I can confirm it fixes the issue. Thanks!
